src/routes/TradingLeague/UserSummary/Percentage/Percentage.tsx

```diff
@@ -0,0 +1,38 @@
+import { Typography } from '@components/atomic';
+import { colors } from '@theme';
+import CountUp from 'react-countup';
+import React from 'react';
+
+type PercentageProps = {
+  percentage: number;
+};
+
+const PercentageTypography: React.FunctionComponent = ({ children }) => (
+  <Typography
+    variant="body2"
+    sx={{
+      color: colors.lavenderWeb.base,
+      fontSize: '14px',
+      lineHeight: '14px',
+      fontWeight: 400,
+    }}
+  >
+    {children}
+  </Typography>
+);
+
+export const Percentage: React.FunctionComponent<PercentageProps> = React.memo(({ percentage }) => {
+  return (
+    <PercentageTypography>
+      <CountUp
```

Comment:
Why do we need a count-up for the progress bar percentage? aren't we overcomplicating things? it seems like this value is loaded instantly.

Reply:
it is nice UI touch. Both progress bar and percentage numbers will animate, giving a nice UX feel. Try the build locally. This has nothing to do with value being loaded instantly. It is just UX touch to make the UI feel more playful

Reply:
I thought this is the actual number i.e. 40% and it was strange to count up to only 40 (or max 100). This loading bar instead looks nice.